applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -872,12 +847,48 @@ bool verify_datamode_control(uint16_t time_limit, uint16_t *min_time_limit)
 	return true;
 }
 
+int slm_get_at_param_list(const char *at_cmd, struct at_param_list **list)
+{
+	int err;
+
+	*list = &at_host_param_list;
+
+	err = at_parser_params_from_str(at_cmd, NULL, *list);
+	if (err) {
+		LOG_ERR("AT command parsing failed: %d", err);
+	}
+
+	return err;
+}
+
+int slm_at_cb_wrapper(char *buf, size_t len, char *at_cmd, slm_at_callback *cb)
+{
+	int err;
+	struct at_param_list *list = NULL;
+
+	assert(cb);
+
+	err = slm_get_at_param_list(at_cmd, &list);
+	if (err) {
```

Comment:
@tomi-font: About: https://github.com/nrfconnect/sdk-nrf/pull/14543#discussion_r1538839431. Github is not doing any favors in locating it. :)

I think that the original comment related to `set_default_at_response`, which no longer exists. If you mean that it should refer to this location, the `slm_get_at_param_list  `

Reply:
> `slm_get_at_param_list` already logs the error.

Ah. Right. All good then.